tools/cldr-code/src/main/java/org/unicode/cldr/test/ConsoleCheckCLDR.java

```diff
@@ -344,6 +344,8 @@ static Level calculateCoverageLevel(final String coverageLevelInput, boolean for
      * @throws Throwable
      */
     public static void main(String[] args) throws Throwable {
+        // turn off logging to not mess up html and other output.
+        CheckCLDR.setLoggerLevel(java.util.logging.Level.OFF);
```

Comment:
instead of disabling the log, how about capturing stderr and stdout separately as in `java ... check ... 2> output.log 1> output.html`?

Reply:
@btangmu the errors are already handled as part of the checks in this case, though.  This doesn't disable stderr.

Reply:
this doesn't suppress any errors, it only suppresses printing out the logging in the ConsoleCheckCLDR tool environment.